test/e2e/e2e.go

```diff
@@ -119,9 +119,6 @@ var _ = ginkgo.SynchronizedAfterSuite(func() {
 }, func() {
 	// Run only Ginkgo on node 1
 	framework.Logf("Running AfterSuite actions on node 1")
-	if framework.TestContext.ReportDir != "" {
-		framework.CoreDump(framework.TestContext.ReportDir)
```

Comment:
`framework.CoreDump` calls `log-dump.sh` to dump cluster logs, but it assumes e2e tests run under kubernetes root.
We do not need this now. `kubetest --dump` does same work.